### ISSUE TITLE
Accessiblity issue with footer "banner" role do not have unique labels

### ISSUE THREAD

**Issue by Stefan8412** [carbon-design-system/carbon-for-ibm-dotcom#9672]
### Description

Hi,
accessibility issue/violations has been found in footer "Multiple elements with "banner" role do not have unique labels after running Accessibility Checker scan.We are using  footer in our project IBM Documentation.

### Component(s) impacted

footer

### Browser

Chrome

### Carbon for IBM.com version

1.32.1

### Severity

Severity 1 = The design is broken in a critical way that blocks users from completing tasks or damages the brand. Affects major functionality, no workaround.

### Application/website

https://www.ibm.com/docs/en/cloud-private

### Package

@carbon/ibmdotcom-react, @carbon/ibmdotcom-styles

### CodeSandbox example

https://www.ibm.com/standards/carbon/web-components/?path=/story/components-footer--default

### Steps to reproduce the issue (if applicable)

  1  Go to Chrome webstore
   2 Look for: IBM Equal Access Accessibility Checker extension
  3  Install the extension and restart browser
   4 open the following link https://www.ibm.com/docs/en/cloud-private
   5 open chrome devtools and choose Accessibility Assessments (next to console/networ/sources/performance)
  6  scan the page
  7  see violation

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by m4olivei** [comment]
This does not appear to be an issue with the footer component itself.

I've put together a CodeSandbox with both @carbon/ibmdotcom-react, @carbon/ibmdotcom-styles at version 1.32.1, with only the footer component to attempt to test this in isolation: https://codesandbox.io/s/loving-wind-w6n65s. When I run the IBM Equal Access Accessibility Checker on this page, there are no violations, and the reported error does not come up.

I can reproduce the error on the reported page,  https://www.ibm.com/docs/en/cloud-private. However, it does not appear to have anything to do with the footer component. The two elements that are showing up with a [banner role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/banner_role) are:

* The `<header>` on the page. This is normal and expected.
* A `<div id="consent_blackbar" role="banner" aria-labelledby="consent_blackbar"></div>` which is showing up a the end of the page.
* You can see this by using [Chrome Dev Tools accessibility tree](https://developer.chrome.com/blog/full-accessibility-tree/):

<img width="1986" alt="dev_tools_accessibilty_tree" src="https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/191049/d32e9b0a-a352-407b-9ef5-226e3e2338dd">

Tangentially, note that the `<footer>` in use here is not being recognized as a landmark role on account of its placement, nested within the `<main>`. That appears to be unintentional, and may be something to address, but is outside the scope of the C4IBM package.
